index.js

```diff
@@ -1,61 +1,55 @@
 module.exports = {
   extends: [
     "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
     "plugin:import/errors",
-    "plugin:react/recommended",
     "plugin:jest/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:promise/recommended",
-    "plugin:sonarjs/recommended"
+    "plugin:react/recommended",
   ],
 
   env: {
     browser: true,
     es6: true,
     jest: true,
-    node: true
+    node: true,
   },
 
   parserOptions: {
     ecmaVersion: 6,
     sourceType: "module",
     ecmaFeatures: {
       jsx: true,
-      modules: true
-    }
+      modules: true,
+    },
   },
 
   parser: "@typescript-eslint/parser",
 
   plugins: [
+    "@typescript-eslint",
     "import",
     "jest",
     "jsx-a11y",
+    "promise",
     "react",
     "react-hooks",
-    "promise",
-    "sonarjs"
   ],
 
   rules: {
```

Comment:
Some eslint rules have been dropped because they are either already in our TS/JS analyzer and activated in our QP or they made in in the recommended config of eslint